### DISCUSSION TITLE
GBA Link compatibility?

### DISCUSSION THREAD

**Discussion by Ashento** [OnionUI/Onion#1621] [Ideas]
Hello, I was wondering if this would ever be implemented in a future OnionOS build. 
The Miyoo Mini Plus does have wi-fi, RetroArch supports it, and even the core TGB Dual already has GB/GBC Link compatibility.

I feel there's plenty of interest in this, as I would myself love to play Pokemon Emerald/FireRed, LoZ:Four Swords, and Sonic Battle with others through our Miyoo Mini Plus devices.

**Comment by jboss38** [comment]
> Hello, I was wondering if this would ever be implemented in a future OnionOS build. The Miyoo Mini Plus does have wi-fi, RetroArch supports it, and even the core TGB Dual already has GB/GBC Link compatibility.
> 
> I feel there's plenty of interest in this, as I would myself love to play Pokemon Emerald/FireRed, LoZ:Four Swords, and Sonic Battle with others through our Miyoo Mini Plus devices.

Hello, I just upload stuff to make it happen, check it out : https://github.com/OnionUI/Onion/pull/1790
